pkg/sdk/poc/generator/helper_methods.go

```diff
@@ -0,0 +1,115 @@
+package generator
+
```

Comment:
`helper_methods` is very generic. A name like `object_helper_methods` is not much better, but I don't have a better idea.

Reply:
did come up with `ResourceHelperMethods` but yeah, its hard to think of a better name